WooCommerce/WooCommerceTests/ViewRelated/Products/ProductListViewModelTests.swift

```diff
@@ -307,4 +307,76 @@ final class ProductListViewModelTests: XCTestCase {
         // Then
         XCTAssertTrue(result.isSuccess)
     }
+
+    // MARK: - Blaze banner
+    func test_updateBlazeBannerVisibility_updates_shouldShowBlazeBanner_to_true_if_site_is_eligible_for_blaze_and_banner_is_not_dismissed_yet() async {
+        // Given
+        let checker = MockBlazeEligibilityChecker(isSiteEligible: true)
+        let uuid = UUID().uuidString
+        let userDefaults = UserDefaults(suiteName: uuid)!
```

Comment:
nit: how about `try XCTUnwrap(UserDefaults(suiteName: uuid))`? (same for other test cases)

Reply:
Updated in 53c389fedf38946f491adce5fe6ff78731a1f2c1.